testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -502,6 +502,13 @@ public void testPartitionedByNestedField()
                         "  TBLPROPERTIES ('format-version'=2)",
                 sparkTableName));
 
+        onSpark().executeQuery(format(
```

Comment:
@zhangjun0x01 lets add unit test-cases as well.

Reply:
The connector doesn't support creating such tables. See https://github.com/trinodb/trino/pull/15712